x-pack/test/alerting_api_integration/common/plugins/actions_simulators/server/bedrock_simulation.ts

```diff
@@ -29,7 +29,7 @@ export class BedrockSimulator extends Simulator {
       return BedrockSimulator.sendErrorResponse(response);
     }
 
-    if (request.url === '/model/anthropic.claude-v2/invoke-with-response-stream') {
+    if (request.url === '/model/anthropic.claude-v2:1/invoke-with-response-stream') {
```

Comment:
Might be good to add a test for the old value here, in case of change in the future where we forget it had a different value ...

Reply:
Not sure if that would be helpful here, this is intercepting the request for response overrides. But I think adding a unit test for `formatBedrockBody` with the old default value makes sense.